spec/views/reports/features.html.erb_spec.rb

```diff
@@ -47,15 +47,33 @@
 
   Page::ANSWER_TYPES.map(&:to_sym).each do |answer_type|
     it "contains a heading for #{answer_type}" do
-      expect(rendered).to have_css("h3.govuk-heading-s", text: answer_type)
+      expect(rendered).to have_css("th", text: answer_type)
     end
 
     it "includes the number of live forms with #{answer_type}" do
-      expect(response.body).to include "Number of forms which ask for this answer type: #{report.live_forms_with_answer_type.attributes[answer_type]}"
+      expect(rendered).to have_css("[data-live-forms-with-answer-type-#{answer_type.to_s.dasherize}]", text: report.live_forms_with_answer_type.attributes[answer_type].to_s)
     end
 
     it "includes the number of live pages with #{answer_type}" do
-      expect(response.body).to include "Number of pages of this answer type: #{report.live_pages_with_answer_type.attributes[answer_type]}"
+      expect(rendered).to have_css("[data-live-pages-with-answer-type-#{answer_type.to_s.dasherize}]", text: report.live_pages_with_answer_type.attributes[answer_type].to_s)
+    end
+  end
+
+  context "when an answer type is missing from the data" do
```

Comment:
nice! I was wondering if this would make it harder to add new answer types.

This test covers that case 👏